docs/guides/connect-aepp-to-wallet.md

```diff
@@ -1,90 +1,109 @@
 # Connect an æpp to a wallet
 
-## Introduction
+This guide describes the 4 steps that are necessary to connect your application to a wallet using the RPC API.
 
-This guide describes the 3 steps that are necessary to connect your application to a wallet using the RPC API.
+## Prerequisites
 
-Note:
-
-- The steps below are snippets taken from the full implementation of the [Simple æpp](https://github.com/aeternity/aepp-sdk-js/tree/master/examples/browser/aepp)
+- Install [Superhero Wallet extension](https://wallet.superhero.com/) for simplicity of example.
```

Comment:
What you think about this added instruction. For the simplicity of example I would advise using Superhero